transcom/mymove
# INT B-19900 move lock feature flag

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19900)

## Summary

In order to set up for the work on [E-04913](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=E-04913), we'll need to have a feature flag to turn on. This PR handles that leg work and setting up the appropriate configs and env variables to prepare for that.

I've also completed and merged a PR into the `milmove-feature-flags` repo: https://github.com/transcom/milmove-feature-flags/pull/128

### How to test

1. In your `envrc`, change `FEATURE_FLAG_MOVE_LOCK` to `true`
2. Run `direnv allow` -> `make server_run` in one terminal
3. Run `make client_run` in another terminal
4. Boot up the admin app and navigate to the feature flags page
5. Throw in `move_lock` and you should get a `true` value that returns. This is what we want - tells us that they found the flag and that it is set to `true`

## Screenshots
![Screenshot 2024-05-06 at 1 51 52 PM](https://github.com/transcom/mymove/assets/136510600/050316ae-43dd-488e-b2fe-ad31ed93b84f)
